exalearn-ui/src/app/app.module.ts

```diff
@@ -32,7 +34,9 @@ const translateRootConfig = {
 		HttpClientModule,
 		TranslateModule.forRoot(translateRootConfig),
 		MainPageModule,
-		LoginPageModule
+		LoginPageModule,
+		MyHistoryPageModule,
```

Comment:
I think we need to move MyHisotryModule and AllHistoryModule to the MainPageModule